code/modules/mod/mod_theme.dm

```diff
@@ -877,6 +877,74 @@
 	acid = 100
 	wound = 15
 
+/datum/mod_theme/praetorian
```

Comment:
Please move this to modular_skyrat/modules/blueshield/code

Reply:
Code made modular as requested